### ISSUE TITLE
Double metadata in Translator

### ISSUE THREAD

**Issue by dohliam** [dohliam/gasp-translator#17]
Moved from an issue reported [here](https://github.com/global-asp/global-asp/issues/13) by @espensj:

> The metadata appears twice in Translator. It can also be a bit spacious, so I suggest making it all on one line. That way the "review" button comes higher up and is more prominent. 
> 
> **Suggestion revision**
> ![revised](https://cloud.githubusercontent.com/assets/12679303/14272314/ded22360-fab6-11e5-812f-833563acda78.png)
> 
> **Current**
> ![screen shot 2016-04-04 at 10 45 57 pm](https://cloud.githubusercontent.com/assets/12679303/14272335/0c32537a-fab7-11e5-801c-4cfb9e22ec1e.png)

**Comment by dohliam** [comment]
Interesting idea... it seems like an easy way to free up some space at the bottom of the page. I'll see what I can do.

**Comment by dohliam** [comment]
Just a note that the metadata is not really "doubled" here -- the section on the left represents the old metadata, and the righthand side shows what the metadata will look like in the new translation (i.e. with a new translator and language).

**Comment by espensj** [comment]
Yes, but in practice they are the same, so it looks confusing to me.
Anyway, not very important.

E
